src/react-contenteditable.js

```diff
@@ -46,11 +54,14 @@ export default class ContentEditable extends React.Component {
     }
   }
 
+  componentWillUnmount() {
+    this.observer.disconnect();
+  }
+
   emitChange(evt) {
     if (!this.htmlEl) return;
     var html = this.htmlEl.innerHTML;
     if (this.props.onChange && html !== this.lastHtml) {
-      evt.target = { value: html };
       this.props.onChange(evt);
```

Comment:
Depends on the node you're getting. In my case I get a text node so `evt.target.innerHTML` isn't a thing. Drop `document.createTextNode('')` in your console to see what's there.